src/display/editor/highlight.js

```diff
@@ -169,11 +169,10 @@ class HighlightEditor extends AnnotationEditor {
         break;
       }
     }
-    const { lastPoint } = this.#focusOutlines.box;
-    this.#lastPoint = [
-      (lastPoint[0] - this.x) / this.width,
-      (lastPoint[1] - this.y) / this.height,
-    ];
+
+    ({ lastPoint: this.#lastPoint } = this.#focusOutlines.box);
+    this.#lastPoint[0] = (this.#lastPoint[0] - x) / width;
+    this.#lastPoint[1] = (this.#lastPoint[1] - y) / height;
```

Comment:
To me this feels a little bit easier to parse if we keep the previous formatting, as below.

Especially given how Objects are passed by reference in JavaScript (since Arrays are just a "special" kind of Object), it seems that with the patch as written we'd be modifying the underlying `this.#focusOutlines.box`-data which we might not actually want here?

```suggestion
    const { lastPoint } = this.#focusOutlines.box;
    this.#lastPoint = [
      (lastPoint[0] - x) / width,
      (lastPoint[1] - y) / height,
    ];
```